clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java

```diff
@@ -95,7 +96,7 @@ public long createEmptyFile(TachyonURI path) throws IOException, InvalidPathExce
    */
   public long createEmptyFile(TachyonURI path, ClientOptions options) throws IOException,
       InvalidPathException, FileAlreadyExistException, BlockInfoException {
-    long fileId = super.create(path, options.getBlockSize(), true);
+    long fileId = super.create(path, options.getBlockSize(), true, Constants.NO_TTL);
```

Comment:
Why not use `options.getTTL()` here? If this is intended, it'll be better to be specified in the javadoc.